docs/operators/how-to/run.md

```diff
@@ -0,0 +1,137 @@
+# Running nwaku
+
+Nwaku binaries can be built and run on Linux and macOS.
```

Comment:
Link to the build doc might be useful

Reply:
[6cb52dc](https://github.com/status-im/nwaku/pull/963/commits/6cb52dcfb28d0b68a1564e7e127172633e056ab2)